src/newrelic_telemetry_sdk/client.py

```diff
@@ -105,11 +114,9 @@ class Client(object):
     PAYLOAD_TYPE = ""
     HOST = ""
     PATH = "/"
-    HEADERS = urllib3.make_headers(
-        keep_alive=True, accept_encoding=True, user_agent=USER_AGENT
-    )
+    HEADERS = urllib3.make_headers(keep_alive=True, accept_encoding=True, user_agent=USER_AGENT)
 
-    def __init__(self, license_key, host=None, port=443):
+    def __init__(self, license_key, host=None, port=443, **connection_pool_kwargs):
```

Comment:
Unrelated to your PR changes: should we be validating that a non-empty license key was supplied here?